CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 * Add ability to retrieve specific Dragon Capsule via `.info('dragon_id')` - [@rodolfobandeira](https://github.com/rodolfobandeira).
 * [#10](https://github.com/rodolfobandeira/spacex/pull/10): Added Ships information - [@zainalmustofa](https://github.com/zainalmustofa).
 * [#9](https://github.com/rodolfobandeira/spacex/pull/9): Added Dragon Capsules information - [@JackieCalapristi](https://github.com/JackieCalapristi).
+* [#7](https://github.com/rodolfobandeira/spacex/pull/7): Added missions information - [@maiafernando](https://github.com/maiafernando).
```

Comment:
Your pull request id is actually `16` instead of `7`.  Please change that. 😄 
7 is the issue id.